src/run-flyway/run-flyway-migrate.ps1

```diff
@@ -18,7 +18,15 @@ $ErrorActionPreference = "Stop";
 
 Write-Information -InformationAction Continue -MessageData "Running migrate..."   
 
-$flywayLocations = "filesystem:`"$(Resolve-Path $pathToMigrationFiles)`""
+$resolvedPaths = New-Object -TypeName "System.Collections.ArrayList"
+$pathToMigrationFiles.Split(",") | ForEach-Object {
+    $resolvedPaths.Add("filesystem:`"$(Resolve-Path $_)`"")
+    Write-Output "resolvedPaths = $resolvedPaths"
```

Comment:
Is there a reason to keep this around since below we're writing out the final list? I imagine it was helpful for debugging but I'm wondering if it would be a little noisy to have happen every time.

Reply:
Removed.